components/_global-footer.scss

```diff
@@ -497,7 +488,8 @@
 			}
 
 			&.wds-is-advertise {
-				height: $desktop-row-bottom-height;
+				margin-right: 0;
```

Comment:
Override defaults. Maybe we could convert those margins into `justify-content: space-between` although I'm not sure if it should be done as a part of this ticket.